lib/Service/Search/FilterStringParser.php

```diff
@@ -119,6 +119,9 @@ private function parseFilterToken(SearchQuery $query, string $token): bool {
 					$query->setEnd($param);
 				}
 				return true;
+			case 'isdefaultsearch':
```

Comment:
```suggestion
			case 'match':
```

and send `anyof` vs `allof`. IMO "default search" is not self explanatory.

Reply:
Needs to be reflected in the remaining code as well. Right now, it is broken because the frontend is still sending `isDefaultSearch` etc.

PS: Spotted the CalDAV nerd ;)